src/openqa_client/client.py

```diff
@@ -377,3 +377,34 @@ def get_jobs(
             # there as it only considers the jobs queried.
             jobdicts = self.find_clones(jobdicts)
         return jobdicts
+
+    def get_latest_build(
+        self,
+        group_id: int,
+        all_passed: bool = True,
+        sorted_key: Callable = float,
+        default_return: str = "0",
+    ) -> str:
+        """Identify latest build number in target Job Group
+
+        Args:
+            group_id (int): Job Group ID
+            all_passed (bool, optional): Controls whether just last build will be selected
+                or the last with all jobs passed. Defaults to True.
+            sorted_key (Callable, optional): To find the latest build we need to order the builds.
```

Comment:
I don't really like the name `sorted_key`, the past tense is odd. I'd call it `sort_key`. The description is a bit odd, it doesn't really explain that this is specifically the key used to sort the results.

Reply:
done

Reply:
>  The description is a bit odd, it doesn't really explain that this is specifically the key used to sort the results.

`The specified callback will be used in the sorted function to sort builds`  :thinking:

Reply:
maybe just `The specified callable will be passed to sorted() to sort the builds.` ?

Reply:
done

Reply:
you are too fast :) I am tweaking code in my working copy and marking conversation done so I know that I solved this one . After I go through ALL long list of endless discussions in this PR I am doing the push . This create a gap in 20 minutes between I wrote "done" and you can actually can confirm that . I was not expecting that you will get to the PR so fast , sorry . NOW it is done